lib/dynamoid/persistence.rb

```diff
@@ -208,7 +197,7 @@ def dynamo_type(type)
               :string
             else
               raise 'unknown type'
-          end
+            end
```

Comment:
Is this intentional?

Reply:
Just my editor formatting the code. I'll update.